### PR TITLE
[MB-11580] Add helper function to get history log display name

### PR DESCRIPTION
**Pull request by jacquelineIO**
## Summary

From [this worksheet](https://docs.google.com/spreadsheets/d/1eaYQadGI-hdENUlR0QApicwkBLJi8lSKrVWPZxILB_0/edit#gid=411873504) we need to map the `operationId` or `eventName` to a human readable name in the history log.

Design:
* [Zeplin](https://app.zeplin.io/project/6216b3c50e72098ae7b8e72e/screen/6216b40a72ff3698e5e62cb3)
* [Refer to the data content miro board to see what needs to be shown](https://miro.com/app/board/o9J_lp2wih8=/?moveToWidget=3458764519189025488&cot=14)

## Setup to Run Your Code

<details>
<summary>💻 You will need to use three separate terminals to test this locally.</summary>

##### Terminal 1

Start the Storybook locally.

```sh
make storybook
```

##### Terminal 2

Start the UI locally.

```sh
make client_run
```

##### Terminal 3

Start the Go server locally.

```sh
make server_run
```

</details>

### Additional steps

1. Access the
2. Login as a
3.

## Verification Steps for Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Request review from a member of a different team.
- [ ] Have the Jira acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Frontend

- [ ] User facing changes have been reviewed by design.
- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
- [ ] There are no new console errors in the browser devtools
- [ ] There are no new console errors in the test output
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/dev/contributing/backend/Backend-Programming-Guide#logging)
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/dev/contributing/backend/Backend-Programming-Guide/#querying-the-database-safely) have been satisfied.

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/dev/contributing/database/Database-Migrations#zero-downtime-migrations)
- [ ] Have been communicated to #g-database
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/dev/contributing/database/Database-Migrations#secure-migrations)

### CODE REVIEW

File: src/shared/historyLogUIDispalyName.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  ['uploadAmendedOrders', 'Updated orders'], //internal.yaml
+  ['submitMoveForApproval', 'Submitted move'], //internal.yaml
+  ['submitAmendedOrders', 'Updated orders'], //internal.yaml
+  ['createMTOShipment', 'Submitted shipments'], //internal.yaml DUPLICATE key
+  ['updateMTOShipment', 'Updated shipments'], //internal.yaml
+  ['createMTOShipment', 'Requested  shipment approval/Shipment request'], //prime.yaml DUPLICATE key
+  ['updateMTOShipment', 'Updated shipment'], //prime.yaml
+  ['updateMTOShipmentAddress', 'Updated shipment'], //prime.yaml
+  ['createMTOServiceItem', 'Requested service item'], //prime.yaml
+]);
```
Comment (rogeruiz) [inline src/shared/historyLogUIDispalyName.js:28]: I wonder if there is a way to generate this programmatically by adding more definitions to the `swagger-def` files which would give us a single place to update things from. But I'm not sure. It's just a theory for now.

Comment (jacquelineIO) [reply src/shared/historyLogUIDispalyName.js:28]: @rogeruiz that's a good idea some sort of constant field, but you won't be able to handle the issue being handled in the switch below https://github.com/transcom/mymove/pull/8270/files#diff-2e2092d1f3ba719e9b3c66ef93590221da17dad450a667969922ade7a15e7a37R92

Comment (rogeruiz) [reply src/shared/historyLogUIDispalyName.js:28]: We're duplicating the `summary` and `description` fields here. We could have the UI Display name come from the `summary` key and leave the `description` field alone.

Here's an example below for one of the `operationIds` with duplicated fields.

[Here's the docs from Swagger on this](https://swagger.io/docs/specification/paths-and-operations/#operations).

```sh
>_ rg -A2 -B3 uploadAmendedOrders -- swagger-def
swagger-def/internal.yaml
2988-    patch:
2989-      summary: Patch the amended orders for a given order
2990-      description: Patch the amended orders for a given order
2991:      operationId: uploadAmendedOrders
2992-      tags:
2993-        - orders
```

**Comment by rogeruiz** [comment]
hey @jacquelineIO is this ready for review now?

**Review by YanZ777 — APPROVED** [review]

**Pull request merged**